src/main/java/org/json/XML.java

```diff
@@ -479,7 +479,7 @@ public static Object stringToValue(String string, XMLXsiTypeConverter<?> typeCon
     // the one in JSONObject. Changes made here should be reflected there.
     // This method should not make calls out of the XML object.
     public static Object stringToValue(String string) {
-        if ("".equals(string)) {
```

Comment:
As this is a public method, the contract has to remain the same. This means supporting a `null` parameter value.